content/en/openpnp/FTP/installing-the-feeders/index.md

```diff
@@ -0,0 +1,152 @@
+  {{< container-image path="images/openPnP-shared/Feeders-tab.png" alt="Switch to the feeders list" >}}
+
+12. Click on the "Add Feeder" icon button.
+  {{< container-image path="images/Add-feeder-button.png" alt="Start adding a new feeder" >}}
+
+13. Select `ReferenceTrayFeeder` and click `Accept`.
+  {{< container-image path="images/Select-referenceTrayFeeder.png" alt="Select and add a new reference tray feeder" >}}
+
+14. In the lower-right panel, you'll be in the `Configuration` tab. Change the `Part` to be `R_0603_1608Metric-R_Small`.
+  {{< container-image path="images/Change-feeder-part.png" alt="Assign the resistor to the tray feeder" >}}
+
+15. Set the `Y` `Offset` to `-4`. This is the space between components on the tape.
+  {{< container-image path="images/Set-feeder-offset.png" alt="Set feeder offset" >}}
+
+16. Set the `Y` `Tray Count` to `30`. This is the number of components available on the feeder before it needs to be manually moved forward.
+  {{< container-image path="images/Set-tray-count.png" alt="Set the feeder tray count" >}}
+
+17. Set the Pick location to: `X=45`, `Y=175`, `Z=20` as a starting point.
+  {{< container-image path="images/Set-left-pick-location.png" alt="Set the starting location for the left feeder" >}}
+
+16. Click on the "Position Camera" icon button to move the top camera roughly over the left feeder.
+  {{< container-image path="images/Position-camera-over-feeder.png" alt="Move the top camera over the left feeder's approximate location" >}}
+
+17. Position the center of the top camera feed over the center of the top-most slot holding a resistor in the tray. You can drag the reticle in the camera feed, or use the jog buttons.
+  {{< container-image path="images/Position-over-feeder-start-rough.png" alt="Manually move closer to the correct feeder location" >}}
+
+18. Zoom in on the camera feed and *precisely* position the center of the reticle over the center of the slot holding the resistor. The resistor itself may not be perfectly centered, that is fine.
+  {{< container-image path="images/Position-over-feeder-start-precise.png" alt="Manually move closer to the correct feeder location" >}}
+
+19. Click the "Capture Camera Location" icon button to save the XY position of the start of the feeder.
+  {{< container-image path="images/Capture-camera-position-feeder.png" alt="Save the current camera position as the start place for the feeder" >}}
+
+20. Click `Apply` to save the feeder settings
+  {{< container-image path="images/Save-feeder-settings.png" alt="Save feeder settings" >}}
+
+21. Click the `Enable` checkbox in the feeder list.
+  {{< container-image path="images/Enable-feeder.png" alt="Enable the feeder so that it is used" >}}
+
+22. Do the same procedure again for the LED feeder. You'll assign the part `LED_0603_1608Metric-LED_Small` to the new feeder.
+
+## Installing the N045 Nozzle
+<!-- TODO: Image of installing N045 nozzle -->
+23. Install a N045 nozzle on your toolhead.
```

Comment:
@sphawes do we need to include a step about lubricating the nozzle holder? If so, can you get photos of that?

Reply:
we've previously mentioned this in the automatic toolchanger section of the docs, but it's definitely worth repeating. i'll get you some pictures!